libs/serialization/include/hpx/serialization/serialize_buffer.hpp

```diff
@@ -95,7 +104,7 @@ namespace hpx { namespace serialization {
             else
```

Comment:
I guess the `boost::shared_array<T>` just two lines above should be `buffer_type` as well?

Reply:
Excellent catch, thanks. Will fix.